### PR TITLE
Start info command

### PR DESCRIPTION
**Pull request by daved**
https://www.pivotaltracker.com/story/show/176004293

 https://www.pivotaltracker.com/story/show/176004293

### CODE REVIEW

File: internal/runners/packages/info.go [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+		if iv.Ingredient.Name != nil {
+			res.name = *iv.Ingredient.Name
+		}
+
+		if iv.Ingredient.Description != nil {
+			res.Description = *iv.Ingredient.Description
+		}
+
+		if iv.Ingredient.Links != nil && iv.Ingredient.Links.Self != nil {
+			res.PkgDetailsTable.Link = iv.Ingredient.Links.Self.String()
```
Comment (Naatan) [inline internal/runners/packages/info.go:116]: This is an API link that shows JSON information. We want the link to the package itself (eg. the github page). If the API doesn't provide this then defer to the AC:

> Any information that's not provided by the search API can be disregarded but should be reported to Pete

File: internal/runners/packages/info.go [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+		}
+	}
+
+	if iv.LatestVersion != nil {
+		if iv.LatestVersion.Version != nil {
+			res.latestVersion = *iv.LatestVersion.Version
+		}
+
+		if iv.LatestVersion.CopyrightText != nil {
+			res.PkgDetailsTable.Copyright = *iv.LatestVersion.CopyrightText
```
Comment (Naatan) [inline internal/runners/packages/info.go:126]: Why is this printing as a JSON array? eg.

```
["Copyright (c) Django Software Foundation and individual contributors."]  
```

File: internal/runners/packages/info.go [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+		}
+
+		if iv.LatestVersion.LicenseExpression != nil {
+			res.PkgDetailsTable.License = *iv.LatestVersion.LicenseExpression
```
Comment (Naatan) [inline internal/runners/packages/info.go:130]: Also printing as a JSON array;

```
["BSD-3-Clause"]  
```

**Review by Naatan — CHANGES_REQUESTED** [review]

File: internal/runners/packages/info.go (hunk 2 above)
Comment (daved) [reply internal/runners/packages/info.go:126]: Discussed in Slack: We're not showing this information to users at this time while changes are made in the backend. (Same for the License field.)

**Review by Naatan — APPROVED** [review]

**Pull request merged**